server/methods/catalog.js

```diff
@@ -613,6 +616,7 @@ Meteor.methods({
     const productId = toDelete[0].ancestors[0];
     toDenormalize.forEach(field => denormalize(productId, field));
 
+    Hooks.Events.run("onProductVariantDelete", deleted);
```

Comment:
`deleted` here is just the results of the `remove` call. I think you want to at least pass in `toDelete` or some info about what product is being deleted